gis/geo_sir/model.py

```diff
@@ -101,7 +104,8 @@ def step(self):
         self.steps += 1
         self.reset_counts()
         self.schedule.step()
-        self.space._recreate_rtree()  # Recalculate spatial tree, because agents are moving
+        self.space._recreate_rtree()  # Recalculate spatial tree,
```

Comment:
The comment should be before the `self.space._recreate_rtree()` line, so that there is continuity in the comment.

Reply:
changes fix this.